micro_sam/sam_annotator/_annotator.py

```diff
@@ -0,0 +1,165 @@
+import numpy as np
+
+from magicgui.widgets import Container, Widget
+
+from . import _widgets as widgets
+from . import util as vutil
+from ._state import AnnotatorState
+
+from typing import TYPE_CHECKING, Optional
+if TYPE_CHECKING:
+    import napari
+
+
+class _AnnotatorBase(Container):
+    """Base class for micro_sam annotation plugins.
+
+    Implements the logic for the 2d, 3d and tracking annotator.
+    The annotators differ in their data dimensionality and the widgets.
+    """
```

Comment:
I noticed you using `@private` in some docstrings elsewhere to stop them appearing in the API docs, should that happen here too? I have no big opinion either way.

Reply:
I think this is not needed here because this is in a file with prefix `_` so it will be ignored by pydoc already.